cid-redirects.json

```diff
@@ -3456,5 +3456,41 @@
   "/docs/integrations/security-threat-detection/ueba": "/docs/integrations/cloud-security-monitoring-analytics",
   "/docs/cse/integrations/automation-service": "/docs/cse/automation-service",
   "/docs/cse/integrations/automation-service-enrichments": "/docs/cse/integrations/enrichments-and-indicators",
+  "/docs/cse/ingestion/auth0": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/auth0",
+  "/docs/cse/ingestion/aws-application-load-balancer": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/aws-application-load-balancer",
+  "/docs/cse/ingestion/aws-cloudtrail": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/aws-cloudtrail",
+  "/docs/cse/ingestion/aws-guardduty": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/aws-guardduty",
+  "/docs/cse/ingestion/aws-network-firewall": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/aws-network-firewall",
+  "/docs/cse/ingestion/aws-vpc-flow": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/aws-vpc-flow",
+  "/docs/cse/ingestion/carbon-black": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/carbon-black",
+  "/docs/cse/ingestion/check-point-firewall": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/check-point-firewall",
+  "/docs/cse/ingestion/cisco-asa": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/cisco-asa",
+  "/docs/cse/ingestion/cisco-meraki": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/cisco-meraki",
+  "/docs/cse/ingestion/corelight-zeek": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/corelight-zeek",
+  "/docs/cse/ingestion/fortigate": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/fortigate-firewall",
```

Comment:
```suggestion
  "/docs/cse/ingestion/fortigate": "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/fortigate",
```

Reply:
@kimsauce, this one has to stay as-is because there is no article for "/docs/cse/ingestion/ingestion-sources-for-cloud-siem/fortigate".